omop_file_validator.py

```diff
@@ -255,6 +272,15 @@ def run_checks(file_path, f):
         ]
         f.seek(0)
 
+        blank_lines = has_blank_lines(f)
+        if blank_lines:
+            blank_lines_msg = 'File contains blank lines.' \
+                              'If there is no data, please only submit the header line.'
+            result['errors'].append(dict(message=blank_lines_msg))
```

Comment:
It would be helpful to see the row numbers for the lines here.

Reply:
Good idea, I'll add them.